src/dll/edbgEcmdDll.C

```diff
@@ -894,8 +791,8 @@ uint32_t queryConfigExistChipUnits(const ecmdChipTarget & i_target, struct pdbg_
       /* Skip targets with no ecmd equivalent */
       continue;
 
-    // If posState is set to VALID, check that our values match
-    // If posState is set to WILDCARD, we don't care
+    //If posState is set to VALID, check that our values match
```

Comment:
Spacing changes shouldn't be part of this PR.  Makes more look like it has changed than actually has.

Reply:
Will fix